helix-term/src/commands/lsp.rs

```diff
@@ -84,17 +84,48 @@ impl ui::menu::Item for lsp::SymbolInformation {
     type Data = Option<lsp::Url>;
 
     fn format(&self, current_doc_path: &Self::Data) -> Row {
-        if current_doc_path.as_ref() == Some(&self.location.uri) {
-            self.name.as_str().into()
-        } else {
-            match self.location.uri.to_file_path() {
-                Ok(path) => {
-                    let get_relative_path = path::get_relative_path(path.as_path());
-                    format!("{} ({})", &self.name, get_relative_path.to_string_lossy()).into()
+        Row::new(vec![
+            Cell::from(match self.kind {
+                lsp::SymbolKind::FILE => "file",
+                lsp::SymbolKind::MODULE => "module",
+                lsp::SymbolKind::NAMESPACE => "namespace",
+                lsp::SymbolKind::PACKAGE => "package",
+                lsp::SymbolKind::CLASS => "class",
+                lsp::SymbolKind::METHOD => "method",
+                lsp::SymbolKind::PROPERTY => "property",
+                lsp::SymbolKind::FIELD => "field",
+                lsp::SymbolKind::CONSTRUCTOR => "constructor",
+                lsp::SymbolKind::ENUM => "enum",
+                lsp::SymbolKind::INTERFACE => "interface",
+                lsp::SymbolKind::FUNCTION => "function",
+                lsp::SymbolKind::VARIABLE => "variable",
+                lsp::SymbolKind::CONSTANT => "constant",
+                lsp::SymbolKind::STRING => "string",
+                lsp::SymbolKind::NUMBER => "number",
+                lsp::SymbolKind::BOOLEAN => "boolean",
+                lsp::SymbolKind::ARRAY => "array",
+                lsp::SymbolKind::OBJECT => "object",
+                lsp::SymbolKind::KEY => "key",
+                lsp::SymbolKind::NULL => "null",
+                lsp::SymbolKind::ENUM_MEMBER => "enum_member",
+                lsp::SymbolKind::STRUCT => "struct",
+                lsp::SymbolKind::EVENT => "event",
+                lsp::SymbolKind::OPERATOR => "operator",
+                lsp::SymbolKind::TYPE_PARAMETER => "type_parameter",
+                kind => unimplemented!("{:?}", kind),
```

Comment:
this will make helix crash if a new LSP spec is released and helix is updated after an LSP. Using `<unknown>` or some other place holder would be better

Reply:
I agree with this, but otherwise, this change looks fine to me 👍 nice and easy quality of life improvement

Reply:
Done, thanks for suggestion!